configs/db/postgres/000001_configuration-management.up.sql

```diff
@@ -53,12 +53,63 @@ CREATE TABLE config_cachedeps (
 
 CREATE INDEX config_cachedeps_idx_cacheid ON config_cachedeps ("cache_id");
 
---DDL for cluster properties:
-CREATE TABLE cluster_props (
-	"id" SERIAL UNIQUE, --just another unique identifer for a property-entry
+--DDL for clusters:
+CREATE TABLE clusters (
+     id uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
 	"cluster" text NOT NULL,
-	"key" text NOT NULL,
-	"value" text NOT NULL,
-	"created" TIMESTAMP WITHOUT TIME ZONE DEFAULT (NOW() AT TIME ZONE 'utc'),
-	CONSTRAINT cluster_metadata_pk PRIMARY KEY ("cluster", "key")
+	"runtime_name" text,
+	"runtime_description" text,
+    "global_account_id" text,
+    "sub_account_id"    text,
+    "service_id"       text,
+    "service_plan_id"   text,
+    "shoot_name"       text,
+    "instance_id"      text,
+	"created" TIMESTAMP WITHOUT TIME ZONE DEFAULT (NOW() AT TIME ZONE 'utc')
+);
+
+CREATE TABLE configurations (
+     id uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
+    "cluster_id" uuid NOT NULL,
+    "kyma_version" text,
+    "kyma_profile" text,
+    "created" TIMESTAMP WITHOUT TIME ZONE DEFAULT (NOW() AT TIME ZONE 'utc'),
+    CONSTRAINT fk_clusters FOREIGN KEY(cluster_id) REFERENCES clusters(id) ON DELETE CASCADE
+);
+
+CREATE TABLE components (
+      "id" uuid PRIMARY KEY CHECK (id <> '00000000-0000-0000-0000-000000000000'),
```

Comment:
Just an idea to make it maybe simpler:we could store the received JSON as string in the configuration entity and parse it on-the-fly when we need it. This would have the benefit, that a could store the contract-version between KEB/Reconciler per configuration-entity which was valid at the time as we received the data. By knowing the contract-version, we can implement contact-specific JSON-hanlder which evaluate the JSON depending on the contract-version.

If we change the contact later, we would only have to add another JSON-renderer version to our application but  we won't have to modify the database structure too.